.github/workflows/optimize-e2e-tests-sm.yml

```
@@ -82,8 +82,19 @@ jobs:
           cache: "yarn"
           cache-dependency-path: optimize/client/yarn.lock
 
-      - name: Build backend & frontend
-        run: mvn clean install -DskipTests -Dskip.docker -pl optimize/backend -am
+      - name: Install node dependencies
+        working-directory: ./optimize/client
+        shell: bash
+        run: yarn
+
+      - name: Build frontend
+        working-directory: ./optimize/client
+        run: yarn build
+        shell: bash
+
+      - name: Start frontend
```

We can potentially run the chain of these commands in the background so the backend build does not have to wait for all the frontend commands. I think it can be achieved like so, but we would have to try it out:
```suggestion
      - name: Build and start frontend
       working-directory: ./optimize/client
        run: { yarn; yarn build; yarn start; } &
        shell: bash
```

---

This won't help much because the job will end up waiting on the "wait for front-end to start" step. 
This will also make the job harder to debug